resources/dicts/events/death/general/warrior.json

```diff
@@ -78,5 +78,13 @@
         "other_cat_skill": null,
         "cat_negate_skill": ["great fighter", "excellent fighter"],
         "other_cat_negate_skill": ["great fighter", "excellent fighter"]
+    },
+    {
+        "camp": "any",
```

Comment:
since this is a death event, it actually won't be available to leaders unless in the leader.json! death events are a lil different like that.  If you'd like this to be able to kill leaders too then you can copy-paste it into the leader.json.  It's good to also add a little sentence to the end that mentions how many lives they lost (in this instance,  one)